css/style.css

```diff
@@ -1,9 +1,379 @@
+}
+
+.d-flex {
+  display: flex;
+}
+
+.f-direction-col {
+  flex-direction: column;
+}
+
+.display-none {
+  display: none;
+}
+
+.f-direction-row {
+  flex-direction: row;
+}
+
+.f-wrap {
+  flex-wrap: wrap;
+}
+
+.j-content-btw {
+  justify-content: space-between;
+}
+
+.j-content-cnt {
+  justify-content: center;
+}
+
+.text-align-center {
+  text-align: center;
+}
+
+.align-items {
+  align-items: center;
+}
+
+.toggle-button {
+  position: absolute;
+  right: 0.72rem;
+  top: 0.4rem;
+  z-index: 2;
+  padding: 8px 16px;
+  background-color: #fff;
+}
+
+.x-button {
+  position: absolute;
+  right: 0.72rem;
+  top: 0.4rem;
+  z-index: 3;
+  background-color: #fff;
+  font-size: 20px;
+  font-weight: bolder;
+  margin-right: 24px;
+}
+
+.bar {
+  transition: 0.4s;
+  background-color: #272a31;
+  display: block;
+  height: 4px;
+  left: 28px;
+  top: 50%;
+  width: 25px;
+}
+
+.bar:first-child {
+  transform: translateY(-4px);
+}
+
+.bar:last-child {
+  transform: translateY(4px);
+}
+
+.nav-ul {
+  flex-direction: column;
+  width: 100%;
+  padding: 16px;
+  position: fixed;
+  z-index: 1;
+  background-color: #fff;
+  top: 0;
+  height: 100vh;
+}
+
+.nav-ul-li {
+  margin: 54px 0;
+  font-size: 17px;
+}
+
+.nav-li-anchor {
+  color: #272a31;
+}
+
+.cc-campaign {
+  border: 2px solid #ec5242;
+  padding: 3px 6px;
+}
+
+.nav-li-anchor:hover {
+  background-color: #000;
+  padding: 12px 17px;
+  color: #fff;
+}
+
+.home-page {
+  /* background-image: url(../images/refugee.jpg); */
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.home-page-first-div {
+  padding: 14px;
+  margin-bottom: 95px;
+}
+
+.home-page-title {
+  font-size: 27px;
+}
+
+.home-page-header {
+  font-size: 37px;
+}
+
+.red-like-color {
+  color: #ec5242;
+}
+
+.article-paragraph {
+  border: 2px solid #fff;
+  line-height: 27px;
+  margin: 34px 0;
+  padding: 4px;
+}
+
+.date-paragraph {
+  font-size: 21px;
+  font-weight: bolder;
+}
+
+.main-program-div {
+  background-color: #000;
+  color: #fff;
+  margin-top: 46px;
+  background-image: url(../images/bacground-image.jpg);
+  background-repeat: round;
+  background-size: contain;
+  width: 100%;
+}
+
+.paragraph-border {
+  border-bottom: 2px solid #ec5242;
+  width: 28px;
+  margin: 10px auto;
+}
+
+.main-program-ul {
+  background-color: #272a31;
+  margin: 4px 22px;
+  align-items: center;
+}
+
+.main-program-ul-li {
+  padding: 34px 8px;
+}
+
+.lecture-image {
+  height: 40px;
+}
+
+.topic {
+  font-size: 19px;
+  color: #ec5242;
+}
+
+.main-program-button-div {
+  padding-bottom: 50px;
+}
+
+.join-button {
+  background-color: #ec5242;
+  opacity: 1;
+  padding: 5%;
+  margin: 23px auto;
+  color: #fff;
+  font-size: 17px;
+  display: block;
+}
+
+.join-button:hover {
+  opacity: 0.8;
+}
+
+.see-program {
+  text-decoration: #fff underline;
+  font-size: 20px;
+  margin: 24px 0;
+}
+
+.join-anchore:hover {
+  opacity: 0.8;
+}
+
+.all-wrappers {
+  margin: 25px 0;
+}

```

Comment:
- [ ] Kindly to match the previous review changes, you can remove the `margin: 25px 0;` to make your design match the original design.

Reply:
Addressed ✔